assets/js/builder.index.entity.databasetable.js

```diff
@@ -257,6 +270,42 @@
         return result
     }
 
+    DatabaseTable.prototype.addIdColumn = function($target) {
+        var existingColumns = this.getColumnNames($target),
+            added = false
+
+        if ($.inArray('id', existingColumns) == -1) {
```

Comment:
@DanHarrin Any particular reason for jQuery's `inArray` over the native `indexOf`?